### PR TITLE
Added Dog model

### PR DESCRIPTION
**Pull request by gerardopg53**

### CODE REVIEW

File: src/main/java/com/Barker/model/Dog.java [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurencyStrategy;
+
+@Entity
+@Table(name = "planet")
```
Comment (corith) [inline src/main/java/com/Barker/model/Dog.java]: planet?

**Review by corith — CHANGES_REQUESTED** [review]

File: src/main/java/com/Barker/model/Dog.java [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Cache;
```
Comment (corith) [inline src/main/java/com/Barker/model/Dog.java:15]: These dependencies need to be brought in through the pom.mxl

**Comment by gerardopg53** [comment]
Requested changes made. Changed table name, added dependencies to pom.xml, added the field sex to dog model, and fixed a typo on an import.

**Comment by gerardopg53** [comment]
bin folder deleted and added to .gitignore.

**Review by corith — APPROVED** [review]

**Pull request merged**